### ISSUE TITLE
":custom-face" not working with multiple faces

### ISSUE THREAD

**Issue by zhengpd** [jwiegley/use-package#600]
I'm trying to custom multiple faces but not working as expected. 

```emacs-lisp
(use-package spaceline-config
  :custom-face
  (powerline-inactive1 ((t (:background nil))))
  (powerline-inactive2 ((t (:background nil))))
  ;; (powerline-active1 ((t (:background nil :foreground nil)))) ; Uncommenting this line gets error
  )
```

This code with two faces works fine, but when I enable the third face line it shows error:

    Error (use-package): Failed to parse package spaceline-config: use-package: spaceline-config wants a (<symbol> <face-spec>) or list of these

Any help would be appreciated.

**Comment by wbolster** [comment]
i think there might be an issue inside `(defun use-package-normalize/:custom-face ...)` in the file `use-package-core.el`.

can you replace this line with the other one and see whether that solves the problem for you?

```patch
@@ -1351,7 +1351,7 @@ no keyword implies `:all'."
             (spec (nth 1 def)))
         (when (or (not face)
                   (not spec)
-                  (> (length arg) 2))
+                  (> (length def) 2))
           (use-package-error error-msg))))))
 
 (defun use-package-handler/:custom-face (name keyword args rest state)
```

**Comment by zhengpd** [comment]
@wbolster I tried your solution and it works. The error is gone and the face takes effect. Thanks.

**Comment by wbolster** [comment]
yw @zhengpd. this should be fixed once #603 is merged.
